test/notifications_test.dart

```diff
@@ -166,11 +167,24 @@ void main() {
         expectedTagComponent: 'stream:${message.streamId}:${message.subject}');
     }));
 
-    test('group DM', () => awaitFakeAsync((async) async {
+    test('group DM: 2 users', () => awaitFakeAsync((async) async {
       await init();
+      final zulipLocalizations = GlobalLocalizations.zulipLocalizations;
       final message = eg.dmMessage(from: eg.thirdUser, to: [eg.otherUser, eg.selfUser]);
```

Comment:
This is a good separate test case to have.

But there are 3 users here, not 2 :slightly_smiling_face: — so the test name should reflect that.

Reply:
Fixed